draft-ietf-ppm-dap.md

```diff
@@ -1245,6 +1250,7 @@ struct {
   TaskID task_id;
   ReportMetadata report_metadata;
   opaque public_share<0..2^32-1>;
+  Extension public_extensions<0..2^32-1>;
```

Comment:
Likewise here: if we include it in report metadata then it will also be covered by AAD.